lib/ui/painting.dart

```diff
@@ -3785,18 +3785,18 @@ class FragmentProgram extends NativeFieldWrapperClass1 {
       spirv,
       spv.TargetLanguage.sksl,
     );
+    if (debugPrint) print(result.src);
```

Comment:
The `print` should go on the next line: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#separate-the-if-expression-from-its-statement

Reply:
Thanks - I meant to delete this actually :)